### PR TITLE
Add the slovenian locale

### PR DESCRIPTION
**Pull request by tipmisle**
This locale is kinda urgent for me, I would be very grateful for a quick review & merge. 
Thank you

### CODE REVIEW

**Review by fhamon — APPROVED** [review]

File: locales/sl.json [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+            },
+            "stock_validation": {
+                "title": "Podatki o zalogi niso na voljo.",
+                "description": "Nekateri izdelki morda niso na zalogi. Posodobite količino."
+            },
+            "custom_fields_validation": {
+                "title": "Nekateri izdelki nimajo vseh obveznih informacij.",
+                "description": "Nekateri izdelki nimajo vseh obveznih informacij. Izpolnite podatke."
+            },
+        },
```
Comment (thatfrankdev) [inline locales/sl.json:313]:
```suggestion
        }
```

File: locales/sl.json [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+            },
+            "stock_validation": {
+                "title": "Podatki o zalogi niso na voljo.",
+                "description": "Nekateri izdelki morda niso na zalogi. Posodobite količino."
+            },
+            "custom_fields_validation": {
+                "title": "Nekateri izdelki nimajo vseh obveznih informacij.",
+                "description": "Nekateri izdelki nimajo vseh obveznih informacij. Izpolnite podatke."
+            },
+        }
```
Comment (thatfrankdev) [inline locales/sl.json:313]:
```suggestion
        },
```

File: locales/sl.json [hunk 3]
```diff
@@ -0,0 +1,376 @@
...
+                "description": "Poskusite jih odstraniti in ponovno dodati v košarico."
+            },
+            "stock_validation": {
+                "title": "Podatki o zalogi niso na voljo.",
+                "description": "Nekateri izdelki morda niso na zalogi. Posodobite količino."
+            },
+            "custom_fields_validation": {
+                "title": "Nekateri izdelki nimajo vseh obveznih informacij.",
+                "description": "Nekateri izdelki nimajo vseh obveznih informacij. Izpolnite podatke."
+            },
```
Comment (thatfrankdev) [inline locales/sl.json:312]:
```suggestion
            }
```

Comment (thatfrankdev) [inline locales/sl.json:312]:
```suggestion
            }
```

**Comment by thatfrankdev** [comment]
A few misclicks later, I ended up getting rid of the extra comma #longWeek :)

Thanks a lot for your contribution @tipmisle

**Comment by tipmisle** [comment]
> A few misclicks later, I ended up getting rid of the extra comma #longWeek :)
> 
> Thanks a lot for your contribution @tipmisle

Thank you for your time and a quick review! I am glad I can contribute.

**Pull request merged**